README.md

```diff
@@ -1 +1,55 @@
-### PORTFOLIO
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Professional personal portfolio website which showcase my personal information, skills and projects as a Fullstack software developer.
+
+![screenshot](./app_screenshot.JPG)
+
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML
+- CSS
+- Webhint
+- Stylelint
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Knowledge on basic html and css
+- Text Editor (VScode, Atom, Sublime text, etc)
+- Basic knowledge of node.js
+
+### Setup
+- clone repository using `git clone`
+- run `npm install`
+- run `live server`
+
+## Authors
+
+👤 **Amedzro Elikplim Emmanuel**
+
+- GitHub: [@githubhandle](https://github.com/Amedzro-Elikplim)
+- Twitter: [@twitterhandle](https://twitter.com/Amedzro-Elikplim)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emmanuel-elikplim-amedzro-187590125/)
```

Comment:
- [ ] [OPTIONAL]: To make your repo professional, you should consider the following:
```
- GitHub: [@Amedzro-Elikplim](https://github.com/Amedzro-Elikplim)
- Twitter: [@Amedzro-Elikplim](https://twitter.com/Amedzro-Elikplim)
- LinkedIn: [Amedzro Elikplim](https://www.linkedin.com/in/emmanuel-elikplim-amedzro-187590125/)
```
> Keeping it as it right now looks very redundant and repetitive.